package.sh

```diff
@@ -1,8 +1,5 @@
 set -eo pipefail
 
-echo Building packages
-pnpm -- lerna run build --stream
-
```

Comment:
build is required? src will be removed later

Reply:
i saw you moved out build to release workflow - but we still need it since `integration tests` is using this file?

Reply:
I also changed that in `integration tests`.
I need to install connectors after building, and before removing, so have to move this command out.